package.json

```diff
@@ -45,6 +45,7 @@
     "@wordpress/data": "2.1.1",
     "@wordpress/date": "2.0.2",
     "@wordpress/deprecated": "2.0.2",
+    "@wordpress/edit-post": "1.0.4",
```

Comment:
Hmm, if we're not really needing this for Calypso but only for our `jetpack-blocks` npm, it should arguably be in https://github.com/Automattic/wp-calypso/blob/87037c078075b7e838146c060069f565161a95d4/client/gutenberg/extensions/presets/jetpack/package.json...

Reply:
I'm not going to block this PR on this since right now, we don't have any dependencies in `jetpack-blocks`' `package.json` anyway. It's something we should do tho (in a separate PR).

Reply:
/cc @sirreal @simison

Reply:
We'll probably be needing this in Calypso soon, so I think it's fine if it remains here.